### ISSUE TITLE
Add parameters that support Chinese encoding.

### ISSUE THREAD

**Issue by ZQFORWARD** [phidatahq/phidata#1483]
When I build an Agent using a tool and the tool returns Chinese characters, sometimes there may be encoding issues that prevent Chinese characters from being recognized correctly. I hope to add a parameter to solve the Chinese encoding problem. 
For example, parameters such as:  json.dumps (content, ensure_ascii=False).

**Comment by ZQFORWARD** [comment]
Of course, the relevant problem points have been identified through debugging. Just add one parameter and modify line 138 in the phi/model/ollama/tools.py file. 
json.dumps ({"name": _fc_message. tool_name, "content": _fc_message.content}, ensure-asci=False)
Chinese output could be correctly

**Comment by ysolanky** [comment]
@ZQFORWARD Thanks for finding the fix! I am going to test this with OllamaTools and integrate this change
